modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,102 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups are also referred to as organizations (or sub organizations). Business groups reside within the master organization and can contain their own business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
+
+[NOTE]
+Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative. There is a limit of 100 business groups per organization.
+
+If you are an organization administrator and your organization is enabled for business group support, you can create business groups within your organization. The organization administrator is also the administrator of any business groups (sub organizations) within the parent organization.
+
+[NOTE]
+When you create a new business group, all the users with the *Organization Administrators* role in the parent business group are also added as organization administrators in the newly created business group (the child business group). However, when you create a new user as an *Organization Administrator* in the parent business group, the user is not added to all the sub business groups.
+
+=== Create a Business Group
+
+Create business groups within your organization. 
+
+[NOTE]
+There is a limit of 100 business groups per organization. 
+
+. In the Anypoint Platform *Access Management* page, in the left menu, click *Organization*.
+. Click the blue plus sign next to the organization name or any of the business groups in the tree diagram.
++
+image::organization-20885.png[]
++
+. In the dialog box, enter:
+  * *Business Group name*: Name for the new business group.
+  * *Owner*: Assign an existing user in the organization to be the business group owner.
+  * Select *Owner can create Business Groups* to allow the owner to create business groups that branch under this business group.
+  * Select *Owner can create environments* to allow the owner to create environments within this business group.
+  * You can assign some or all of the vCores that your organization owns to an individual business group. This ensures that these are used by the CloudHub deployments within the business group. You can assign these when creating the business group or edit these settings later.
+  * *Static IPs*: Enable the use of static IPs.
+. Click *Add Business Group*.
+
+image::organization-9fa5f.png[]
+
+[NOTE]
+--
+Allocating vCores to a business group makes those vCores available only to that business group, which makes them unavailable to the parent organization.
+--
+
+==== Child Business Groups
+
+You can create hierarchical levels of business groups to provide more control over access. To create a child business group click the blue *+* icon next to any business group.
```

Comment:
This task, even though short, should probably have its own heading, and be among the tasks, not here in the middle of a conceptual section?